vllm/entrypoints/openai/rpc/server.py

```diff
@@ -3,6 +3,7 @@
 from typing import Any, Coroutine
 
 import cloudpickle
+import uvloop
```

Comment:
Is it already required by another package? It wouldn't hurt to explicitly add it to requirements-common.txt

Reply:
It's already installed via uvloop which is a dependency of fastapi i beleive

Reply:
[`fastapi` recently excluded `uvicorn` as a dependency](https://github.com/fastapi/fastapi/releases/tag/0.112.0) but `uvicorn[standard]` (which depends on `uvloop`) is already in vllm's `requirements-common.txt` so it is safe for now.